Shared/alerts/AlertSource.h

```diff
@@ -17,8 +17,9 @@
 #ifndef ALERTSOURCE_H
 #define ALERTSOURCE_H
 
-// C++ API headers
-#include "Point.h"
+namespace Esri::ArcGISRuntime {
+class Point;
```

Comment:
Minor nitpick. Some of these are indented and others aren't. Choose one (my vote is indented). If this will add more time and codebase churn, leave it.